docs/diagrams/AddPersonDobActivityDiagram.puml

```diff
@@ -0,0 +1,19 @@
+@startuml
+start
+repeat
+:User enters add person command;
+if () then ([date of birth field is provided])
+  if () then ([date of birth format is valid])
+    if () then ([person doesn't exist
+   in the contact list])
```

Comment:
need to refactor `contact list` to `persons list`, same for this line in `AddPersonGenderActivityDiagram.puml`
```suggestion
   in the persons list])
```

Reply:
yup done